### DISCUSSION TITLE
Bot reading server specific nicknames

### DISCUSSION THREAD

**Discussion by Krattsk** [gameserverapp/Platform#647] [Suggestions]
Ability for the GSA bot to read nicknames in the Discord server and then using that nickname in cluster chat when chatting from Discord to in-game server would be great to have. :)
A nice Discord icon instead of [Discord] would be a pretty upgrade too.

**Comment by mvaessen** [comment]
for reference https://discord.com/channels/315049169071112193/424240875732860929/1184225968945102939

**Comment by mvaessen** [comment]
Nickname usage has been added to cluster chat (duplicate item)
